src/ompl/base/spaces/DubinsStateSpace.h

```diff
@@ -71,6 +71,26 @@ namespace ompl
                 DUBINS_STRAIGHT = 1,
                 DUBINS_RIGHT = 2
             };
+
+            enum DubinsClass
```

Comment:
I'd move this to DubinsStateSpace.cpp inside an anonymous namespace, since it is only used internally and not in the public API.

Reply:
Done!